Cargo.toml

```diff
@@ -220,7 +220,7 @@ memchr = "2.4.1"
 memmap2 = "0.5.0"
 memoffset = "0.6.4"
 multimap = "0.8.2"
-nix = "0.22"
+nix = { version = "0.26", features = ["fs", "resource", "signal"] }
```

Comment:
Needed to bump this for access to `nix::sys::resource` (added in 0.23) and `resource::RLIM_INFINITY` (added in 0.26).

Reply:
I updated this to match what we use internally, you should be able to drop it if you rebase on master